subworkflows/local/utils_nfcore_ascc_pipeline/main.nf

```diff
@@ -135,7 +135,7 @@ workflow PIPELINE_INITIALISATION {
     standardised_unzipped_input
         .branch{
```

Comment:
always good to name inputs to operators - then change in the below lines `it[0].assembly_type` to `meta.assembly_type`

(am just assuming structure here)

```suggestion
        .branch{ meta, fasta -> 
```